### ISSUE TITLE
Dev Server wont start by npm start

### ISSUE THREAD

**Issue by guygolanIL** [facebook/react-360#768]
Might be related to issue #316 . 
Following the react 360 official tutorial i got stuck when starting the dev server by running npm start.

Bug
Followed the installation tutorial step by step from the official docs.
1. npm install -g react-360-cli
2. react-360 init Hello360
3. cd Hello360
4. npm start
and this is the output:
PS C:\Users\guy.golan\Desktop\hello-react-360> npm start        

hello_react_360@0.0.1 start C:\Users\guy.golan\Desktop\hello-react-360
node node_modules/react-360/scripts/packager.js

open browser at http://localhost:8081/index.html

Starting React Native Packager...
┌──────────────────────────────────────────────────────────────────────────────┐
│                                                                              │
│  Running Metro Bundler on port 8081.                                         │
│                                                                              │
│  Keep Metro running while developing on any JS projects. Feel free to        │
│  close this tab and run your own Metro instance if you prefer.               │
│                                                                              │
│  https://github.com/facebook/react-native                                    │
│                                                                              │
└──────────────────────────────────────────────────────────────────────────────┘

Looking for JS files in
   C:\Users\guy.golan\Desktop\hello-react-360

Metro Bundler ready.

Loading dependency graph, done.

It looks like its stuck after the "Loading dependency graph, done." statement.

Expecting the server to be up at http://localhost:8081/index.html but no response.

### Additional Information

* 'react-360' package version: [~1.1.0]
* 'react-360-web' package version: [~1.1.0]
* Operating System: [Windows]
* Browser: [Chrome]
* VR Device: [no set]

**Comment by ansible-comme** [comment]
Any reason there's a difference between your init `Hello360` and folder name `hello-react-360` in your comment?

Anything showing in the console when you open localhost after the "Loading dependency graph, done." statement.?

**Comment by guygolanIL** [comment]
> Any reason there's a difference between your init `Hello360` and folder name `hello-react-360` in your comment?

Yeah sorry for the confusion, i tried creating multiple projects with the init command. Its Irrelevant since the init process for both project was the same. ill edit the main post.

> Anything showing in the console when you open localhost after the "Loading dependency graph, done." statement.?
After 'Loading dependency graph, done.' there is no other line.
Honestly it looks like its "--watch"ing  for changes.

**Comment by guygolanIL** [comment]
I saw that React 360/VR has some connection to React Native so it might be worthy to note that i have never installed anything related to React Native.

**Comment by ansible-comme** [comment]
> Yeah sorry for the confusion
np thought that might be the case

How about your browser console, does it log any errors there? Or what does the localhost response say in browser?

**Comment by guygolanIL** [comment]
> How about your browser console, does it log any errors there? Or what does the localhost response say in browser?

In the browser (Chrome) I dont event reach the server I get "This page isn't working" ERR_EMPTY_RESPONSE

**Comment by ansible-comme** [comment]
Some general suggestions but I've limited experience...

- update node
- run `npm start -- --reset-cache`
- VPN could be an issue
- conflict from different package you've used in the past (?)
- clear browser cache (worked for me with http-server issue)
- Refresh your browser after "Loading dependency graph, done." to kick start the bundle process.

**Comment by guygolanIL** [comment]
Hi. I went and updated my node to version 12.13.1 and now executing npm start is throwing this output:

PS C:\Users\user\Desktop\Hello360> npm start

> Hello360@0.0.1 start C:\Users\user\Desktop\Hello360
> node node_modules/react-360/scripts/packager.js

open browser at http://localhost:8081/index.html


An error occurred during the packager process. Exited with code 1. d character class 
Look at the packager output above to see what went wrong.
npm ERR! code ELIFECYCLE                                                             dules\\.*|heapCapture\\bundle\.js|.*\\__tests__\\.*)$/: Unterminated character class     
npm ERR! errno 1
npm ERR! Hello360@0.0.1 start: `node node_modules/react-360/scripts/packager.js`
npm ERR! Exit status 1                                             /packager.js`     
npm ERR!
npm ERR! Failed at the Hello360@0.0.1 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.                                    dditional logging 

npm ERR! A complete log of this run can be found in:                                 output above.
npm ERR!     C:\Users\user\AppData\Roaming\npm-cache\_logs\2019-12-09T17_22_59_866Z-debug.log                                         09T17_22_59_866Z-d
PS C:\Users\user\Desktop\Hello360> node -v                                           ebug.log
v12.13.1
PS C:\Users\user\Desktop\Hello360>

**Comment by ansible-comme** [comment]
Some issues with React Native w/ Expo too, possible fix here: https://github.com/expo/expo-cli/issues/1074#issuecomment-546167583

**Comment by sprucetonye** [comment]
> Some issues with React Native w/ Expo too, possible fix here: [expo/expo-cli#1074 (comment)](https://github.com/expo/expo-cli/issues/1074#issuecomment-546167583)

but React 360 does not have  metro-config\src\defaults\blacklist.js so where can we possibly get this help or should i format my system? thanks

**Comment by sprucetonye** [comment]
Hey I'm sorry about the misconception. **Cheers**

**Comment by cskbv** [comment]
Hi, I get the same error too and still have not been able to fix it. Can someone please help me do it 

An error occurred during the packager process. Exited with code 1.
Look at the packager output above to see what went wrong.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! Hello360@0.0.1 start: `node node_modules/react-360/scripts/packager.js "--reset-cache"`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the Hello360@0.0.1 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\karti\AppData\Roaming\npm-cache\_logs\2020-06-15T05_57_19_324Z-debug.log



This is the Error I am getting can someone please try and help me again

**Comment by sprucetonye** [comment]
Hi @cskbv 
Can down grade your node version to 10.1 
But please ensure you've uninstalled node completely before doing the own grade.

**Comment by deepakkhanna18** [comment]
@guygolanIL guygolanIL

Any solution you got for it getting stuck on "It looks like its stuck after the "Loading dependency graph, done." statement."

My Hello360 basic example program getting stucked on this

I am using windows 10

Kindly helo
